README.md

```diff
@@ -26,18 +26,11 @@ The Ingress workflow is as follows:
   cloud storage, and puts a message on the announcement topic while applying a header
   to identify the destination service.
 
-For the vast majority of topics, the above process is accurate. For payloads from both
+For the vast majority of upload types, the above process is accurate. For payloads from both
 **ansible tower** and the **insights cluster operator**, the flow is a bit different. The key difference here is that it relies on the UHC Auth Proxy for authentication.
 
 ![UML](https://www.plantuml.com/plantuml/png/TP5FSvim4CNl-HHRNzBEDEP0Jpt5mTF6qwRrXFYSMSC6D0Y9QbTntKzV2Mpe4FV4_dc_vV6uPK4dljLNxvGfj2y9Qf6EFoU9myEoKbBxlMToXJL2HfQ5RPDEeudC3KkfrJx9FjriusZty3rfaOLS63rdWK1bo2sxUFygFuPD-pwy92e-mY8RgaKeDuPLLGl3puuSYdMB3sSzDjYY2ffLNqHrjlunxLCkK5EOfdaiudwrtS1N53hWSTBvkWYhtNq6AoyrR9tzVOpYs94HLQ0eQo0dzweAcZXbAaVCqUHGHMZNQOlbMp6BTLZHdRDD_udvqCCmY6IUdfeBSDhlUrCj_h46xhGj6ZWVcO17qbEEOq23gTxV_TFJDX-4puzZX6DKNwmxe2jXZqmbM0Daoiug8pDs33U6Duzg9Xbp6g-BFG-11-lpyoF3wHHgXyVuZ3WBLa43Uryq9F-dvwcJAQ4Dgp2CPzx-XM_uqk3fp8pkhJpOL_hNoBLrrMQTd38FbQDVdbWswsjG1cn7Xclr8XUStWOpljL_0G00)
 
-### Kafka Topics (Legacy)
```

Comment:
Along with this you also need to remove the reference to "legacy" in the `Content Type` section:
> In the case of the legacy operation, the service name tells ingress what topic to use:
>
>application/vnd.redhat.advisor.example+tgz => platform.upload.advisor

Reply:
:+1: